### ISSUE TITLE
nqtail multi user output 

### ISSUE THREAD

**Issue by RTLhamster** [leahneukirchen/nq#49]
I tried nq and nqtail in a user envirionment to sequentially run programs.
The issue I see is that the other user can see the program output (via nqtail) of the job started from the other user.

user1:
nq command1
nqtail -> sees his output

user2
nq command2
nqtail -> sees output of command1 and then command2

Is there a way that the user only sees the output of his own jobs?

**Comment by leahneukirchen** [comment]
You can patch that in by checking the uid of the file creator, but you cannot stop users from seeing the logs, as the log file itself needs to be readable.

**Comment by RTLhamster** [comment]
yes, its clear that the files can be seen by all users since these files are readable by all users.
My concern is more the user experience, if the user sees the output of others: its confusing.
By default I cannot pass the UID to nqtail, or am I wrong?
